zerver/views/messages.py

```diff
@@ -1276,6 +1276,15 @@ def get_message_edit_history(request: HttpRequest, user_profile: UserProfile,
         return json_error(_("Message edit history is disabled in this organization"))
     message, ignored_user_message = access_message(user_profile, message_id)
 
+    if message.edit_history is None:  # There is no history for this message
+        return json_success({'message_history': [{
+            'user_id': message.sender_id,
+            'rendered_content': message.rendered_content,
+            'timestamp': message.pub_date,
+            'content': message.content,
+            'topic': message.subject
+        }]})
```

Comment:
Why not just pass `[]` to `fill_edit_history_entries` instead?  Seems like it would produce the same result and be more maintainable.

Reply:
I already tried that. The problem is that `fill_edit_history_entries` assumes that the history isn't empty, so if we pass an empty list as argument this assertion will fail:

```py
assert(datetime_to_timestamp(message.last_edit_time) == message_history[0]['timestamp'])
```

I'm not sure why we need that type of assertion there (after all it's unhandled, if it fails the server will just show a 500), so if we can think of a better way to make that check, like checking out the length of `message_history` —or just getting rid of it— I can update this.

Reply:
Let's just move that assertion into an `if len(message_history) > 0:`.  I think it is helpful for detecting if we end up with corrupted data in the edit history data structure.